core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -195,11 +196,21 @@ public boolean onOptionsItemSelected(MenuItem item) {
       onBackPressed();
       return true;
     } else if (itemId == R.id.menu_bookmarks_clear) {
-      presenter.deleteBookmarks(new ArrayList<>(allBookmarks));
-      allBookmarks.clear();
-      bookmarksList.clear();
-      bookmarksAdapter.notifyDataSetChanged();
-      Toast.makeText(this, R.string.all_bookmarks_cleared_toast, Toast.LENGTH_SHORT).show();
+      new AlertDialog.Builder(this)
+              .setTitle(getResources().getString(R.string.clear_all_bookmark_dialog_title))
```

Comment:
Activities have a method `getString` so you don't need to use `getResources` to get a string but also there is a version of `setTitle` that takes an int